client/src/services/loginHelper.js

```diff
@@ -1,16 +1,16 @@
-import axios from 'axios'
+import axios from "axios";
 
 /**
  * @function loginHelper
  * @param {str ing} email a- e-mail t=_hat comes from the form
  * @param {string} password - password that comes from the form
  */
 
-export default  async function loginHelper (email,password){
-    const response = await axios.post("/login", {
-        email: email,
-        password:password
-    })
+export default async function loginHelper(email, password) {
+  const response = await axios.post("http://localhost:5000/login", {
```

Comment:
This is now a hardcoded path the localhost. If we go to production this line will still refer to localhost. Do you already have an idea on how you would like to do this?

Reply:
I believe this one is already changed in the development branch and will merge from there :)